### PR TITLE
Bug 1975714: Add policy-group label to the openshift-console namespace manifest

### PR DESCRIPTION
**Pull request by jhadvig**
Adding the `network.openshift.io/policy-group` label based.
@sleshchenko FYI

/assign @spadgett

### CODE REVIEW

File: manifests/02-namespace.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     openshift.io/node-selector: ""
     workload.openshift.io/allowed: "management"
+    network.openshift.io/policy-group: "console"
```
Comment (sleshchenko) [inline manifests/02-namespace.yaml:11]: Well, I'm checking on mobile but AFAIS you add annotation but label is needed on our end

**Comment by jhadvig** [comment]
@sleshchenko comment addressed. PTAL

**Review by sleshchenko — APPROVED** [review]
LGTM

**Comment by sleshchenko** [comment]
Please also backport to OS 4.7, since it blocks installing WTO on DevSandbox https://issues.redhat.com/browse/CRT-938

**Review by spadgett — APPROVED** [review]
/lgtm
/hold
@jhadvig Can you attach this to a Bugzilla? Then we can remove the hold.

**Pull request merged**